app/src/views/RootLayout/index.tsx

```diff
@@ -56,6 +62,12 @@ export function Component() {
 
     const [languagePending, setLanguagePending] = useState(false);
 
+    const [isCookiesBannerVisible, setIsCookiesBannerVisible] = useState(true);
```

Comment:
Don't we save information that user has accepted the cookie consent? What happens when user does not accept cookie consent